eve/main.yml

```
@@ -1924,27 +1925,32 @@ stages:
           command: >
               major=$(echo "%(prop:product_version)s" | cut -d'.' -f1) &&
               minor=$(echo "%(prop:product_version)s" | cut -d'.' -f2) &&
-              git tag --sort=taggerdate --list "$major.$(( $minor-1 )).*" | tail -1
+              git tag --sort=taggerdate --list "$major.$(( $minor-1 )).*" | grep -v '\-' | tail -1
```

Just so you know, you can also use `--` option to provide a pattern starting with a `-`:
```suggestion
              git tag --sort=taggerdate --list "$major.$(( $minor-1 )).*" | grep -v -- - | tail -1
```

---

Ohh ok good to know :+1:, but I will not do it here right now (do not want to wait for another build) :smile: